appauthlibrary/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# If your project uses WebView with JS, uncomment the following
+# and specify the fully qualified class name to the JavaScript interface
+# class:
+#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
+#   public *;
+#}
+
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
```

Comment:
Should we keep this on?  The final app assembly (customer app) can obfuscate if they want. (i'm not sure what the defaults are, but it seems to imply this does some obfuscating?) 🤷‍♂️

Reply:
That is a default generated proguard rules, I will remove it as we don't need to have it at all )